cordova-lib/src/hooks/scriptsFinder.js

```diff
@@ -105,6 +105,11 @@ function getApplicationHookScriptsFromDir(dir) {
  */
 function getScriptsFromConfigXml(hook, opts) {
     var configPath = cordovaUtil.projectConfig(opts.projectRoot);
+
+    if (!(fs.existsSync(configPath))) {
```

Comment:
a comment here that lets code readers know that there are cases when the cordova project structure is not available would be helpful.